src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,38 @@
+.stars {
+  display: flex;
+  gap: 4px;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  display: flex;
+  /* margin-left: 4px; */
+}
+
+/* .stars__star:first-child {
+  margin-left: 0;
+} */
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(../images/star-active.svg);
```

Comment:
Ensure that the image path '../images/star-active.svg' is correct and that the image file exists at this location.